docs/module_docs/featdet_adap_chromatogram_builder/adap-chromatogram-builder.md

```diff
@@ -1,33 +1,35 @@
-# ADAP chromatogram builder
+# **ADAP chromatogram builder**
+The _ADAP chromatogram builder_ module is one of the LC-MS feature detection algorithms provided by MZmine 3. The module essentially builds an [EIC](../../terminology/general-terminology.md#extracted-ion-chromatogram) for each _m/z_ value that was detected over a minum number of consecutive scans in the LC-MS run.
+Each data file is processed individually. The [mass list](../../terminology/general-terminology.md#mass-list) associated to each MS1 scan in a data file (see [Mass detection](../featdet_mass_detection/mass-detection.md) module) are taken as input and a [feature list](../../terminology/general-terminology.md#feature-list) is returned as output. Since a mass list must be avaialble, the _Mass detection_ module must be run first.
 
-![adap chromatogram builder](adapchromatogrambuilder.png)
+The _ADAP chromatogram builder_ algorithm operates as follow. Only MS1 scans are processed. All the the data points are extracted from all the MS1 scans in a data file and and sorted in order of decreasing intensity. The processing starts from the most intense data point and, since no EICs have yet been created, a new EIC is initialized and associated to the corresponding _m/z_ value. The processing proceeds with the second-highest data point and the corresponding _m/z_ is checked to determine if it "belongs" to the existing EIC based the user-defined tolerance (_i.e._ "Scan to scan accuracy (m/z)" parameter). If so, the data point is added to the EIC and the EIC-associated _m/z_ is updated. Otherwise, a new EIC is inizialized. When no new data point can be added to an EIC, the latter is terminated. The process is iterad until all the data points have been processed and a set of EICs has been created (constituting the _feature list_). Finally, the EICs are checked according to the user-defined parameters (_i.e._ minimum number of data points and intensity). The EICs matching the requirements are retained in the feature list, whereas the rest are discarded. The so-built EICs can then be resolved into individual features by one of the deconvolution algorithms provided by MZmine 3 (_e.g._ [Local mimimum resolver](../featdet_resolver_local_minimum/local-minimum-resolver.md) module).
```

Comment:
delete: When no new data point can be added to an EIC, the latter is terminated.
this is not the case. you can think of the created eic, like a list that has a spot reserved for every MS1 scan. If a data point is found for the scan, the spot is taken up by the data point. otherwise 0 intensity is assumed. The building does not terminate if there is a missing data point
(but this does not have to be in the docs, it's jsut background knowledge)

iterad -> iterated